sympy/core/backend.py

```diff
@@ -12,6 +12,28 @@
         ImmutableMatrix, MatrixBase, Rational, Basic)
     from symengine.lib.symengine_wrapper import gcd as igcd
     from symengine import AppliedUndef
+
+    def sympify(obj, *, strict=False):
+        """SymEngine-compatible strict ``sympify``.
```

Comment:
Wouldn't it be better to, instead of overwriting the `sympify` docstring, add a notes section to the `sympify_symengine.__doc__` that the kwarg `strict` has been implemented?

Reply:
Addressed in https://github.com/sympy/sympy/pull/25116/commits/fdb929f08ea2bd7d938c99038a892b9187ea0bd3.